DQM/L1TMonitor/python/L1TCSCTF_cfi.py

```diff
@@ -17,5 +17,6 @@
 #
 # Make changes for running in Run 2
 #
-from Configuration.StandardSequences.Eras import eras
-eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
+# This is not working.
+#from Configuration.StandardSequences.Eras import eras
+#eras.run2_common.toModify( l1tCsctf, gangedME11a = False )
```

Comment:
@deguio @davidlange6 
Okay, now I see get the solution. This config:
https://github.com/cms-sw/cmssw/blob/CMSSW_7_4_X/SLHCUpgradeSimulations/Configuration/python/postLS1Customs.py#L133

is totally not used in our online DQM framework, so the flag is never set correctly.
We have two solutions, one is to make the change what we have in this PR. Second is to add  
SLHCUpgradeSimulations config, but this looks like for "simulation" from its name.  Let me know what should we do.